package excecao;

public class Causa {

	public static void main(String[] args) {

		try {
			MetodoA(null);
		} catch (Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}

	}

	static void MetodoA(Aluno aluno) {
		try {
			MetodoB(aluno);
		} catch (Exception cause) {
			throw new IllegalArgumentException(cause);
		}
	}

	static void MetodoB(Aluno aluno) {
		if (aluno == null) {
			throw new NullPointerException("ALUNO ESTA NULLO!!!");
		}
		System.out.println(aluno.nome);
	}

}
